app-crypt/signify/signify-23.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="Cryptographically sign and verify files"
+DESCRIPTION="OpenBSD tool to signs and verify signatures on files. \
+	Portable version"
```

Comment:
Sorry, I forgot to mention this one. No need for a tab in line 3.
Everything else is OK. I'll validate your GCI task and leave this PR open for a couple of days in case we have good news from https://dev.haiku-os.org/ticket/13869